common/persistence/tests/queue_v2_test_suite.go

```diff
@@ -533,4 +661,14 @@ func testListQueues(ctx context.Context, t *testing.T, queue persistence.QueueV2
 		})
 		assert.Error(t, err)
 	})
+	t.Run("InvalidMetadataEncoding", func(t *testing.T) {
```

Comment:
Is this an invalid metadata encoding or an invalid page token?

Reply:
Looks like I copy-pasted this test and forgot to remove. This test is not needed. For Cassandra, invalidMetadataEncoding is tested in testCassandraQueueV2ErrInvalidPayloadEncodingType()
For SQL, we are using an already-tested code path to get metadata. Removing this test from here.